ui/main/src/app/views/customCardList/ResultTable.spec.ts

```diff
@@ -557,13 +557,62 @@ describe('CustomScreenView - ResultTable', () => {
                     id: 'id3'
                 })
             ];
-            resultTable.setTypesOfStateFilter(['INPROGRESS', 'FINISHED']);
+            resultTable.setTypesOfStateFilter([TypeOfStateEnum.INPROGRESS, TypeOfStateEnum.FINISHED]);
             const dataArray = resultTable.getDataArrayFromCards(cards, emptyChildCardsList);
             expect(dataArray).toEqual([
                 {cardId: 'id0', testField: 'processId0'},
                 {cardId: 'id1', testField: 'processId0'}
             ]);
         });
+        it('by read or ack', () => {
```

Comment:
Should have more test , i suggest to create a describe and a before that init the resultTable and init the card list